wsclient/src/main/java/org/hpccsystems/ws/client/wrappers/gen/filespray/AbortDFUWorkunitWrapper.java

```diff
@@ -20,11 +20,11 @@
 
 /**
  * Generated Axis2 ADB stub class wrapper
- * WrapperMaker version: 1.7
+ * WrapperMaker version: 1.8
  * Class name: AbortDFUWorkunitWrapper
  * Wraps class: org.hpccsystems.ws.client.gen.axis2.filespray.latest.AbortDFUWorkunit
  * Output package : org.hpccsystems.ws.client.wrappers.gen.filespray
- * Service version: 1.23
```

Comment:
What are your thoughts on potentially removing the service version here? I think it would make these interface change commits a lot cleaner in the future, in that only the wrappers that have changes would be present in the commit. I think this could be beneficial in understanding what has changed in between service versions.

Reply:
I definitely would like to prevent unnecessary diffs, but the corresponding service version seems useful, even if the class didn't change.